crates/rooch-indexer/src/types.rs

```diff
@@ -156,27 +158,27 @@ impl IndexedEvent {
 
 #[derive(Debug, Clone)]
 pub struct IndexedGlobalState {
-    /// The global state key
+    // The global state key
     pub object_id: ObjectID,
-    /// The owner of the object
```

Comment:
`///` on the field should be ok?

Reply:
@jolestar you're right. During the replacement, an inadvertent substitution occurred simultaneously.